cadence/contracts/GametvNFT.cdc

```diff
@@ -183,8 +181,17 @@ pub contract GametvNFT: NonFungibleToken {
     pub resource NftTypeHelper : NftTypeHelperPublic {
         // public function to borrow details of NFTtype
         pub fun borrowNFTtype(id: String): NftType? {
-            if GametvNFT.nftTypes[id] != nil {
-                let ref = GametvNFT.nftTypes[id]
+            if GametvNFT.activeNftTypes[id] != nil {
+                let ref = GametvNFT.activeNftTypes[id]
+                return ref
+            } else {
+                return nil
+            }
+        }
+
+        pub fun borrowStaleNFTtype(id: String): NftType? {
```

Comment:
Why not name this borrowHistoricNFTtype given the dict name is historicNftTypes? Let's also follow a single convention for the string 'NftType' - it's either NftType everywhere or NFTtype, preferably the former. Please correct other instances in the code as well.

Reply:
Already did it few days back.